ParameterDataApi.yaml

```
@@ -143,60 +199,69 @@ components:
     HeaderDefinition:
       type: object
       properties: 
-        startTime:
-          description: The start time for the time series
-          type: string
-          format: date-time
-        
-        endTime: 
-          description: The end time for the time series
+        messageType:
+          type: string          
+          description: The type of the message.
+
+        version:
           type: string
-          format: date-time
+          description: The version string for the parameter data messageType.
           
-        columns:
-          $ref: '#/components/schemas/ColumnDefinition'
-        
       required:
-        - startTime
-        - endTime
+        - messageType
+        - version
+
+    SingleDataDefinition:
+      type: object
+      description: A definition of a single parameter data sample. This is loosely based upon Json Time Series, Version 1.0. (https://docs.eagle.io/en/latest/reference/index.html)
+      properties:
+
+        header:
+          $ref: '#/components/schemas/HeaderDefinition'
+          
+        columns:
```

Does "single parameter data sample" equate to a single column?
```suggestion
        column:
```